build.gradle

```diff
@@ -3,6 +3,7 @@ buildscript {
         if (System.getenv('VERIFY_USE_PUBLIC_BINARIES') == 'true') {
           logger.warn('Production builds MUST NOT be built with public binaries.\nUse artifactory/whitelisted-repos for production builds.\n\n')
           maven { url 'https://dl.bintray.com/alphagov/maven-test' }
+          maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases' }
```

Comment:
Part of me thinks that this shouldn't be necessary ... should our dependencies be published somewhere ... I'll have a look in the team manual.

Reply:
Jakub said that Travis builds use the public repositories.

Reply:
Slight confusion on my part, this seems to be only used for the Travis builds, so its OK.